controller/gui/template/network.mustache

```diff
@@ -36,6 +51,13 @@
 
             <pre class="d-Preformatted">{{properties}}</pre>
 
+
+            <form action="/network/{{id}}/proxy" method="post" class="d-Network__Form">
+              <legend class="d-Network__Legend">Proxy</legend>
+              <input type="text" class="d-Input d-Network__Input" value="{{proxy}}" name="proxy_url" placeholder="192.168.1.10:8500">
+              <input type="submit" class="d-Button" value="Validate">
+            </form>
```

Comment:
Maybe this should be moved underneath the `connected` guard? It is a bit confusing to have two forms (with the proxy form appearing first) on a disconnected wifi. In addition, trying to configure a proxy on a disconnected wifi does not actually do anything, but claims to have been successful.

Reply:
Done with https://github.com/dividat/playos/pull/44/commits/87aa57eb5f0916767de6825ac93570199426fdb1.

I just tested setting a proxy with an unconnected wifi:
- after having applied it, it does not show the proxy.
- but once we are connected to the network, it show it.

So, it works in the end, but the application is not giving an easy feedback.

Maybe we should only enable the proxy form to the connected network.

Reply:
Proxy form opening with a toggle:

![2020-10-28_15-43-39](https://user-images.githubusercontent.com/6768842/97453424-02277200-1936-11eb-8567-473d4472e92b.gif)

Reply:
> Maybe we should only enable the proxy form to the connected network.

Yes, either that or fuse everything into one form to be submitted together (but the latter is more effort-full).

Reply:
Fusing the forms seems to be a really good idea, done in https://github.com/dividat/playos/pull/44/commits/69f6da0e2dced2cc8897f5e1c3308f29ba0c6278.

![2020-10-30_15-41-13](https://user-images.githubusercontent.com/6768842/97718608-6bd68600-1ac6-11eb-866b-7fe9d76baba7.png)

If we need the user to specify ipv4 or another field in the future, it can be extended easily.

Proxy information is still readable in the orange block.